### ISSUE TITLE
How to run this example at Duo S ?

### ISSUE THREAD

**Issue by Sfinx** [milkv-duo/duo-8051#1]
https://milkv.io/docs/duo/getting-started/8051core instructions do not work

**Comment by yxiaoa** [comment]
meet the same question, have you solved it?

**Comment by jagspaul** [comment]
yes it works for me on DUO 64M but partial. It does not works for dw_timer.c , also timer interrupt not working.

Please you have to modify the ini file with correct address on which your 8051 bin file going to be loaded.

jags
